.env.template

```diff
@@ -4,3 +4,10 @@ FLASK_ENV=development
 
 # Change the following values for local development.
 SECRET_KEY=secret-key
```

Comment:
This line can also go now (since we've moved away from the flask session cookie)
```suggestion
```